src/Module.php

```diff
@@ -16,4 +16,12 @@ final class Module extends \luya\base\Module implements CoreModuleInterface
      * visible pages are included by default. In order to include hidden pages into the sitemap enable $withHidden.
      */
     public $withHidden = false;
+
+    /**
+     * @var boolean Whether the sitemap URLs should be encoded or not. Its highly recommend to enable this property as otherwise
+     * the Sitemap file could have errors and therfore won't work.
+     * @since 0.X.0
```

Comment:
@SOHELAHMED7 @cebe I was not sure which version will be released, therefore you should change this before or afterwards :-)